internal/pkg/fixtures/fixtures.go

```diff
@@ -14,7 +14,8 @@ import (
 )
 
 type ProjectSnapshot struct {
-	Branches []*BranchConfigs `json:"branches"`
+	Branches  []*BranchConfigs  `json:"branches"`
```

Comment:
@webrouse Takovej detail, klidně to fixnu. Neměla by se ta struktura jmenovat spíš jen `BranchConfig` v jednotným čísle, když `Branches` jsou už slice nad ní?

Reply:
su to `configS` z `branch` :) tj. tych konfiguracii je tam viac ... ta Slice zase znamena, ze je viac `branch`:
![image](https://user-images.githubusercontent.com/19371734/139858807-e7681d71-4af4-4be0-b01a-055a72571794.png)

Idealne by sa to volalo asi `Branch`, ale tu potrebujeme aj bez `Configs`, teda uz existuje.

Reply:
`BranchWithConfigs`?

Reply:
Jo tak! 😄 Já myslel že to znamená konfigurace jedné konkrétní branche (ve který jsou konfigurace komponent). Ale `BranchWithConfigs` je imho lepší, upravím to teda. 👍

Reply:
:+1: